src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -232,17 +240,49 @@ public PackageInfo BuildModel(string id, XElement element)
             });
         }
 
+        private void EnsureValidFindPackagesResponse(XDocument doc)
+        {
+            if (!string.Equals(doc.Root.GetDefaultNamespace().ToString(), _defaultNamespace,
+                StringComparison.OrdinalIgnoreCase))
```

Comment:
A error page can be a valid XML file, so we use the default namespace to tell whether it is a valid NuGet service response